tests/testNWBFile.cpp

```diff
@@ -48,5 +52,32 @@ TEST_CASE("createElectricalSeries", "[nwb]")
 
   nwbfile.finalize();
```

Comment:
```suggestion
  Status finalized nwbfile.finalize();
  REQUIRE(finalized == Status::Success);
```

It would also be useful to have some checks that the data that is being written here is correct, or that being checked elsewhere?

Reply:
I have some checks for content as part of the `testNWBRecording.cpp` and some general checks for data block write in `testHDF5IO.cpp`. However it's probably a good idea to add additional checks for each data type.

Reply:
I forgot that I had added checks for `ElectricalSeries` in `testEcephys.cpp`. I think that should cover these cases for now.